### DISCUSSION TITLE
Any idea how to nest a child and use x-slot?

### DISCUSSION THREAD

**Discussion by EriksonEnterprises** [livewire/livewire#8907] [1. Help]
I have a parent "global bootstrap modal" that lets me do cool things like calling a component within the global bootstrap modal component 

I was hoping I could somehow utilize x-slot to further simplify my coding so that I can have a child component with a simple `<x-slot name="title">` and then in the parent it'll have `{{ $title }}`

But I can't figure out how to get that to work as it seems like the child component is never going to push the x-slot properly to the parent component 

Any ideas?

**Comment by lrljoe** [comment]
Check out the Wire Elements Modal package for dynamically loading  Livewire Modal components 

Otherwise, sharing a more detailed example will help others help you

**Reply by EriksonEnterprises** [reply]
Yeah I was trying to do that but unfortunately that project doesn't allow @push and scripts (like ckeditor type stuff)

So I was just going to do a global modal instead but seems like I'll have to just have each full modal in each component and won't be able to pass slots lol
